addons/website_blog/i18n/website_blog.pot

```diff
@@ -363,6 +363,11 @@ msgstr ""
 msgid "All blogs"
 msgstr ""
 
+#: code:addons/website_blog/static/src/js/website_blog.js:32
```

Comment:
It's not line 32 ;)

Reply:
Whoops, forgot to change that, fixed!

Reply:
Not sure how this is to be managed forward-porting though :shrug: probably not important I hope... but for major versions and since the PR had to be modified, I guess better safe than sorry :shrug: